Robot_Adapter/CRUD/Update/Loads/LoadCombinations.cs

```diff
@@ -42,7 +44,78 @@ public partial class RobotAdapter
         protected bool Update(IEnumerable<LoadCombination> loadCombinations)
         {
             bool success = true;
-            success = ICreate(loadCombinations);
+            
+            m_RobotApplication.Project.Structure.Cases.BeginMultiOperation();
+            
+            foreach (LoadCombination lComb in loadCombinations)
+            {
+                //Check combination itself is not null
+                if (!CheckNotNull(lComb))
+                    continue;
+
+                // Use the Number property directly and try to get the combination
+                int combinationId = lComb.Number;
+                
+                // Check if the RobotId matches the LoadCombination Number
+                int robotId = GetAdapterId<int>(lComb);
+                
+                // If the LoadCombination doesn't have a RobotId, assign it from the Number
+                if (robotId == 0)
+                {
+                    this.SetAdapterId(lComb, combinationId);
+                    robotId = combinationId;
+                    Engine.Base.Compute.RecordNote($"LoadCombination with number {combinationId} did not have a RobotId. RobotId has been set to the LoadCombination number.");
+                }
+                else if (robotId != combinationId)
+                {
+                    Engine.Base.Compute.RecordError($"Load combination has mismatched IDs: RobotId = {robotId}, Number = {combinationId}. Using Number property for update.");
+                }
+                
+                // Get the existing combination from Robot (following pattern from Loadcases Update method)
+                RobotCaseCombination rCaseCombination = m_RobotApplication.Project.Structure.Cases.Get(combinationId) as RobotCaseCombination;
+                if (rCaseCombination == null)
+                {
+                    Engine.Base.Compute.RecordWarning("Load combination with number " + combinationId.ToString() + " does not exist in Robot. Load combination could not be updated!");
+                    success = false;
+                    continue;
```

Comment:
```suggestion
                    Engine.Base.Compute.RecordError("Load combination with number " + combinationId.ToString() + " does not exist in Robot. Load combination could not be updated!");
                    return false;
```
Suggest we return an error here and exit the method.